### PR TITLE
en guide improvement

### PR DESCRIPTION
**Pull request by AnnStepanova**
@AleksandrHorev 
@kkkhesha

### CODE REVIEW

**Review by kkkhesha — APPROVED** [review]

File: README-en.md [hunk 1]
```diff
@@ -10,23 +10,24 @@ All changes to the strings you make in your own repository without affecting the
...
+1. Open [Surf for Translators](https://ton-surf-translate.firebaseapp.com/). If opening for the fist time, please comply with legal requirements as discribed in the [Help Center](https://help.ton.surf/en/support/solutions/articles/77000236686-do-i-need-to-register-to-use-surf-).
 2. In the upper left corner, click the Surf icon.
 3. Scroll down and find the Surf current version number.
    ![alt text](images/screenshot-19.png)
-4. On GitHub, open your local repositiory copied from TON-Surf-Localization.
-5. Click on the branch name and select the branch of the same version you found at [Surf for Translators](https://ton-surf-translate.firebaseapp.com/).
-6. If there is no appropriate branch in the list, create it the following way:
+4. Open the bookmark with your local repositiory copied from TON-Surf-Localization.
+5. Click on the branch name and enter the version number you found at [Surf for Translators](https://ton-surf-translate.firebaseapp.com/).
+   ![alt text](images/screenshot-23.png)
```
Comment (AleksandrHorev) [inline README-en.md:29]: image

File: README-en.md [hunk 2]
```diff
@@ -35,20 +36,28 @@ To get the file latest version, please check the current Surf version number and
 
 ## Open file for edit
 
-You can edit files right in GitHub or download them to your computer and edit locally.
+You can edit files right in GitHub or download them to your computer and edit locally. All localization files are stored in the **src** folder.
+![alt text](images/screenshot-24.png)
```
Comment (AleksandrHorev) [inline README-en.md:40]: image

File: README-en.md [hunk 3]
```diff
@@ -67,6 +76,9 @@ Sometimes different strings can have the same translation. To know exactly what
 Now, when you put the cursor in the string, its ID is displayed above the screen.
 ![alt text](images/screenshot-05.png)
 
+Every ID consists of keys separated with commas. Knowing the string ID you can search the localization file for keys and find the string you need. 
+![alt text](images/screenshot-25.png)
```
Comment (AleksandrHorev) [inline README-en.md:80]: image

File: README-en.md [hunk 4]
```diff
@@ -111,11 +123,13 @@ To receive notifications when strings changed and you need to take part in revie
 1. In the upper-right corner of the page, click your profile photo, then select **Settings**.
    ![alt text](images/screenshot-11.png)
 2. In the **Account settings** menu, select **Notifications**.
-3. ![alt text](images/screenshot-12.pnge)
+3. ![alt text](images/screenshot-12.png)
```
Comment (AleksandrHorev) [inline README-en.md:126]: image

**Review by AleksandrHorev — APPROVED** [review]

**Pull request merged**